tests/js/spec/views/organizationDiscover/discover.spec.jsx

```diff
@@ -116,12 +117,24 @@ describe('Discover', function() {
   describe('reset()', function() {
     let wrapper, queryBuilder;
     beforeEach(function() {
+      browserHistory.push = function(url) {
+        wrapper.setProps({
```

Comment:
I think by default `browserHistory.push` is a jest mock fn, so you could use `browserHistory.push.mockImplementationOnce(fn)`.

I think what would be helpful to test here is: click Reset button, assert that `push` gets called, assert `queryBuilder.reset()` gets called, and then we should also check that state gets reset?

Reply:
Added expectations for state variables and `queryBuilder.reset()`